common/app/layout/Front.scala

```diff
@@ -269,7 +165,7 @@ object FaciaContainer {
     // popular containers should never be sponsored
     container match {
       case MostPopular => ContainerCommercialOptions.mostPopular(omitMPU)
```

Comment:
Do we still need `mostPopular` and `empty` given that they each just wrap `ContainerCommercialOptions`?